docs/src/dev/developer/development-environment.asciidoc

```diff
@@ -248,6 +248,23 @@ can be used to help run tests.
 
 See the <<release-environment,Release Environment>> section for more information on release manager configurations.
 
+[[go-environment]]
+=== Go Environment
+
+The build optionally requires link:https://go.dev/dl/[Go] (>=1.7) to work with the `gremlin-go` module. If Go is not installed, TinkerPop will still build with Maven, but Go projects will be skipped.
```

Comment:
Version > 1.17.

Reply:
fixed in https://github.com/lyndonbauto/tinkerpop/pull/83/commits/1001ef64e2ae1df6ee8813ea5dbeac2f4bff0a19